concepticondata/references/references.bib

```diff
@@ -1705,7 +1705,13 @@ @Book{Matisoff1978
   Title                               = {{V}ariational semantics in {T}ibeto-{B}urman. {T}he 'organic' approach to linguistic comparison},
   Author                              = {Matisoff, James A.},
   Year                                = {1978},
-  Publisher                           = {Institute for the Study of Human Issues}
+  Publisher                           = {Institute for the Study of Human Issues},
+  Pages                               = {xviii+331},
+  Address                             = {Occasional Papers of the Wolfenden Society on Tibeto-Burman Linguistics, Volume VI. Philadelphia},
+  Publisher                           = {Philadelphia: Institute for the Study of Human Issues},
```

Comment:
Two publishers here, I think this should be reduced to one, right?

Reply:
done